test_othermlpackages/test_tensorflow.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+from . import package_test_common as ptc
+
+from fairlearn.reductions import DemographicParity
+
+from tensorflow.keras.layers import Dense
+from tensorflow.keras.models import Sequential
+from tensorflow.keras.wrappers.scikit_learn import KerasClassifier
+
+
+def create_model():
+    # create model
+    model = Sequential()
```

Comment:
This is largely copied from a tutorial online, so please let me know if there's something better to do.

Reply:
The structure of the test and the model architecture seem to be fine, but what is this test actually checking? I think you're showing that even with a KerasClassifier, which in turn is a wrapper wrapping a network with fully connected layers and cross entropy loss, the run completes. However, the test would be more robust if you could get the returned model and assert that the shape, of the output tensor after every layer in the neural net, is as expected.

Reply:
What would the output tensor after every layer be expected to be? Sorry, I'd not touched TensorFlow before this morning.

I have just added some extra tests in the common file which verify that the disparities have been reduced as expected.

Reply:
Could use a combination of tf.get_layer and tf.shape to assert on the shape of a given layer, or it could also be some simple assertions like the ones mentioned here:
https://www.tensorflow.org/api_docs/python/tf/keras/layers/Layer

Not necessary, but only if you'd want to add any tf specific unit testing.

Reply:
I could add that later, but I don't want to test TensorFlow itself - I want to make sure that TensorFlow can be used with Fairlearn.